openhcl/openhcl_boot/src/arch/x86_64/tdx.rs

```diff
@@ -98,6 +163,21 @@ impl minimal_rt::arch::IoAccess for TdxIoAccess {
     }
 }
 
+pub fn invoke_tdcall_hypercall(
```

Comment:
I think this function also needs to be unsafe, because data will be written to input_page and output_page, and the caller must specify valid addresses for those.